pyvenv.cfg

```diff
@@ -0,0 +1,3 @@
+home = /Users/tdivoll/.pyenv/versions/3.9.7/bin
```

Comment:
what is this file? Seems user specific. I don't remember seeing it in my other peen/poetry projects

Reply:
That looks much better!

Reply:
remove this file